src/mongo/db/pipeline/expression_test.cpp

```diff
@@ -2864,6 +2864,24 @@ TEST(ExpressionObjectOptimizations, OptimizingAnObjectShouldOptimizeSubExpressio
     ASSERT_VALUE_EQ(expConstant->evaluate(Document()), Value(3));
 };
 
+TEST(ExpressionObjectOptimizations, OptimizingAnObjectWithAllConstantsShouldOptimizeToExpressionConstant) {
+
+    intrusive_ptr<ExpressionContextForTest> expCtx(new ExpressionContextForTest());
+    VariablesParseState vps = expCtx->variablesParseState;
+
+    //All constants should optimize to ExpressionConstant
```

Comment:
It looks like you haven't run clang-format on this code. Comments should be separated from the `//` by a space. Also, use a full sentence here, and end with a period.
`// Test that an object with constant values for every field will optimize to a constant value.`